packages/usa-select/src/usa-select--multiple.twig

```diff
@@ -0,0 +1,13 @@
+<form class="usa-form">
+  <label class="usa-label" for="options">Dropdown label</label>
+  <select class="usa-select" name="options" id="options" {% if multiple %} multiple  size={{ size }} {% endif %} >
+    <option value>- Select -</option>
+    <option value="value1">Option A</option>
+    <option value="value2">Option B</option>
+    <option value="value3">Option C</option>
+    <option value="value1">Option E</option>
+    <option value="value2">Option F</option>
+    <option value="value3">Option G</option>
+  </select>
+</form>
+
```

Comment:
Can we move this to a directory `usa-select/test/test-patterns`?

Similar to what we did with this component test template
https://github.com/uswds/uswds/blob/1db0e68d7e5cf21ae52f2d8a29d5963c7aa1b681/packages/usa-combo-box/src/test/test-patterns/test-usa-combo-box.twig